CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+Styles v3.0.0
+------
+- introduce dependency on a spritesheet:
+		- example for `light` theme https://protomaps.github.io/basemaps-assets/sprites/v3/light`
+- Change townspots from separate symbol layer to sprite layer
```

Comment:
Since the style's complexity / sophistication is increasing, maybe the following text to help untangle it for people who are following along?

> - Migrate town labels to single "symbol" style layer to specify linked "icon" and "text", using the new spritesheet to source the icon's image. This resolves a bug where townspot icons in the old `places_locality_circle` "circle" style layer were often orphaned (still displayed) even if the related text in the `places_locality` "symbol" style layer couldn't be placed. The circle style layer has been removed and consolidated into the symbol style layer.

Reply:
Updated!